mmcv/runner/iter_based_runner.py

```diff
@@ -161,6 +161,20 @@ def resume(self,
         self._epoch = checkpoint['meta']['epoch']
         self._iter = checkpoint['meta']['iter']
         self._inner_iter = checkpoint['meta']['iter']
+
+        # Raising warning when resuming models with different number of GPUs
```

Comment:
add a comment to explain why we recommend setting the same number of GPUs for resuming